src/scss/_main.scss

```diff
@@ -1,4 +1,4 @@
 body {
-  width: 100%;
-  background-color: red;
```

Comment:
Zostaw proszę, to na początek, do momentu, aż nie zmienimy tła.

Reply:
Już przywróciłem te dwie linijki.